examples/tutorial/tutorial_notebooks/introduction/omnisci_on_native/exercise_2.ipynb

```diff
@@ -0,0 +1,383 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![LOGO](../img/MODIN_ver2_hrz.png)\n",
+    "\n",
+    "<center><h2>Scale your pandas workflows by changing one line of code</h2>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 2: Speed improvements\n",
+    "\n",
+    "**GOAL**: Learn about common functionality that Modin speeds up by using all of your machine's cores."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Concept for Exercise: `read_csv` speedups\n",
+    "\n",
+    "The most commonly used data ingestion method used in pandas is CSV files (link to pandas survey). This concept is designed to give an idea of the kinds of speedups possible, even on a non-distributed filesystem. Modin also supports other file formats for parallel and distributed reads, which can be found in the documentation.\n",
+    "\n",
+    "![](../img/read_csv_perf.png)"
```

Comment:
We should put read_csv_perf.png here that is related to OmniSci.

Reply:
@YarShev this was changed